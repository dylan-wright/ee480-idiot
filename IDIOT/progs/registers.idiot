and $zero, $zero
and $one, $one
and $sign, $sign
and $all, $all
and $sp, $sp
and $fp, $fp
and $ra, $ra
and $rv, $rv
and $u0, $u0
and $u1, $u1
and $u2, $u2
and $u3, $u3
and $u4, $u4
and $u5, $u5
and $u6, $u6
and $u7, $u7
and $u8, $u8
and $u9, $u9
and $u10, $u10
and $u11, $u11
and $u12, $u12
and $u13, $u13
and $u14, $u14
and $u15, $u15
and $u16, $u16
and $u17, $u17
and $u18, $u18
and $u19, $u19
and $u20, $u20
and $u21, $u21
and $u22, $u22
and $u23, $u23
and $u24, $u24
and $u25, $u25
and $u26, $u26
and $u27, $u27
and $u28, $u28
and $u29, $u29
and $u30, $u30
and $u31, $u31
and $u32, $u32
and $u33, $u33
and $u34, $u34
and $u35, $u35
and $u36, $u36
and $u37, $u37
and $u38, $u38
and $u39, $u39
and $u40, $u40
and $u41, $u41
and $u42, $u42
and $u43, $u43
and $u44, $u44
and $u45, $u45
and $u46, $u46
and $u47, $u47
and $u48, $u48
and $u49, $u49
and $u50, $u50
and $u51, $u51
and $u52, $u52
and $u53, $u53
and $u54, $u54
and $u55, $u55
