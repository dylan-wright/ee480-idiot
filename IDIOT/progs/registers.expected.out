Text:
//generated by AIK version 20070512
@0000
1000
1041
1082
10c3
1104
1145
1186
11c7
1208
1249
128a
12cb
130c
134d
138e
13cf
1410
1451
1492
14d3
1514
1555
1596
15d7
1618
1659
169a
16db
171c
175d
179e
17df
1820
1861
18a2
18e3
1924
1965
19a6
19e7
1a28
1a69
1aaa
1aeb
1b2c
1b6d
1bae
1bef
1c30
1c71
1cb2
1cf3
1d34
1d75
1db6
1df7
1e38
1e79
1eba
1efb
1f3c
1f7d
1fbe
1fff
//end
Data:
//generated by AIK version 20070512
//end
